src/main/resources/liquibase/crop_changelog/studies_TBD.xml

```diff
@@ -0,0 +1,642 @@
+				add generation_level int null comment 'cross expansion (pedigree) level';
+		</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="studies-TBD-20">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT count(1) > 0 FROM projectprop WHERE variable_id = 8360
+            </sqlCheck>
+        </preConditions>
+        <comment>Replace the SEED_SOURCE variable by ENTRY_SOURCE IN projectprop AND stockprop</comment>
+        <sql dbms="mysql" splitStatements="true">
+			# Replace SEED_SOURCE variable by ENTRY_SOURCE
+			UPDATE projectprop SET variable_id = ( SELECT cvterm_id FROM CVTERM WHERE NAME ='ENTRY_SOURCE' ), ALIAS = 'ENTRY_SOURCE', TYPE_ID = 1815
+				WHERE variable_id = 8360 AND cv_id=1040 AND is_obsolete = 0;
+			# Update cvterm_id 8360 from (SEED_SOURCE) to ENTRY_SOURCE cvterm_id
+			UPDATE stockprop SET TYPE_ID = ( SELECT cvterm_id FROM CVTERM WHERE NAME ='ENTRY_SOURCE' )
+				WHERE TYPE_ID = 8360;
+        </sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="studies-TBD-21">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) from cvterm where cvterm_id = 8360 and cv_id = 1040;
+				</sqlCheck>
+		</preConditions>
+		<comment>
+			Remove SEED_SOURCE variable from cvterm table and from cvterm_relationship table
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM cvterm where cvterm_id = 8360;
+			DELETE FROM cvterm_relationship where subject_id = 8360;
+			DELETE FROM cvtermprop where cvterm_id = 8360;
+		</sql>
+	</changeSet>
+
+	<changeSet author="lkovacic" id="studies-TBD-22">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT count(1) > 0 FROM stockprop WHERE type_id = 8330;
+			</sqlCheck>
+		</preConditions>
+		<comment>Delete group gid values in 'stockprop' table</comment>
+		<sql dbms="mysql" splitStatements="true">
+ 			DELETE FROM stockprop WHERE type_id = 8330;
+		</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="studies-TBD-23">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(1) > 0 FROM stockprop WHERE type_id = 8378;
+            </sqlCheck>
+        </preConditions>
+        <comment>Delete values of 'SOURCE' variable from stockprop table</comment>
+        <sql dbms="mysql" splitStatements="true">
+             DELETE FROM stockprop where type_id = 8378;
+        </sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="studies-TBD-24">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) from cvterm where cvterm_id = 8378;
+            </sqlCheck>
+        </preConditions>
+        <comment>Rename 'SOURCE' as 'IMMEDIATE_SOURCE_NAME' in cvterm and projectprop tables</comment>
+        <sql dbms="mysql" splitStatements="true">
+             UPDATE cvterm SET name = 'IMMEDIATE_SOURCE_NAME' WHERE cvterm_id = 8378;
```

Comment:
@cuenyad we should use type_id in where clause to make sure we just modify aliases for Germplasm descriptors types